packages/react-devtools-shared/src/backend/renderer.js

```diff
@@ -220,7 +220,7 @@ export function getInternalReactConstants(version: string): {
       HostSingleton: 27, // Same as above
       HostText: 6,
       IncompleteClassComponent: 17,
-      IndeterminateComponent: 2,
+      IndeterminateComponent: 2, // removed in 19.0.0
```

Comment:
I think better way is to add an extra `gte(version, '19.0.0')` check at the top of this `if / else` statements, which will return the same `ReactTypeOfWork`, but with `IndeterminateComponent: -1 // Removed`.

Reply:
I originally did this but Seb pointed out, and I think this makes sense, that we really only need to create a new config by version when we modify the number used for a certain kind of component. In 19 there simply will be no IndeterminateComponent's in the tree but we don't really need to enforce this via devtools so we can just mark this as removed in a comment and if we have some other motivation to clean this up in the future we can get rid of it then.

It'd be interesting as an exercise to potentially collapse the config to be only when the tag numbers changed, it might make the tool slightly smaller and we'd have fewer branches for this tag map